draft-ietf-quic-recovery.md

```diff
@@ -59,12 +44,7 @@ normative:
 
 informative:
 
-  RFC3782:
```

Comment:
Correct, but these are all relevant informative references, so I'd rather keep them.  The intent is to reference them before the next version.